<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lEPath">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDuplicates_1">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_editcopy.png</normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_editcopy.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_Finder_1">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_viewmag.png</normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_viewmag.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="filesExplorerView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>16</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lEPath_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbDuplicates_2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_editcopy.png</normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_editcopy.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_Finder_2">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_viewmag.png</normaloff>:/FinderIcons/LGPLIcons/Crystal_Clear_action_viewmag.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="filesExplorerView_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>16</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbOpen">
         <property name="text">
          <string>F&amp;3 Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCompare">
         <property name="text">
          <string>F&amp;4 Compare</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCopy">
         <property name="text">
          <string>F&amp;5 Copy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbMove">
         <property name="text">
          <string>F&amp;6 Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbMkdir">
         <property name="text">
          <string>F&amp;7 Mkdir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRemove">
         <property name="text">
          <string>F&amp;8 Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRename">
         <property name="text">
          <string>F&amp;9 Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbExit">
         <property name="text">
          <string>F10 &amp;Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbAbout">
         <property name="text">
          <string>F12 &amp;About</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="FileChartDockView" name="chartDockWidget_1">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>150</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="FileChartDockView" name="chartDockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>150</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileChartDockView</class>
   <extends>QDockWidget</extends>
   <header>filechartdockview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filesExplorerView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>chartDockWidget_1</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filesExplorerView_2</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>chartDockWidget_2</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
